examples/80_extending/example_restrict_number_of_hyperparameters.py

```diff
@@ -0,0 +1,134 @@
+"""
+===================================================================
+Restricting the number of hyperparameters for an existing component
+===================================================================
+
+The following example demonstrates how to replace an existing
+component with a new component, implementing the same classifier,
+but with different hyperparameters .
+"""
+
+from ConfigSpace.configuration_space import ConfigurationSpace
+from ConfigSpace.hyperparameters import UniformIntegerHyperparameter, UniformFloatHyperparameter
+
+from sklearn.datasets import load_breast_cancer
+from sklearn.model_selection import train_test_split
+
+import autosklearn.classification
+import autosklearn.pipeline.components.classification
+from autosklearn.pipeline.components.classification \
+    import AutoSklearnClassificationAlgorithm
+from autosklearn.pipeline.constants import DENSE, UNSIGNED_DATA, PREDICTIONS, SPARSE
+
+
+############################################################################
+# Subclass auto-sklearn's random forest classifier
+# ================================================
+
+# This classifier only has one of the hyperparameter's that auto-sklearn's
```

Comment:
Do you think it sounds better if it is changed to: This classifier only has one of the hyperparameter's *of* auto-sklearn's ...

Reply:
Yes, I think so.